content/en/continuous_integration/guides/flaky_test_management.md

```diff
@@ -26,6 +26,8 @@ The app helps you prioritize flaky tests by providing the following information
 
 Once you identify a flaky test you want to fix, click on the test to see links to view the most recent failed test run or the first flaky test run.
 
+<div class="alert alert-info"><strong>Note</strong>: The table is limited to the 1000 flaky tests that exhibited flakiness in more commits for the selected time window.</div>
```

Comment:
```suggestion
<div class="alert alert-info"><strong>Note</strong>: The table is limited to the 1000 flaky tests that exhibited flakiness for the selected time window.</div>
```

Reply:
Hi @maycmlee, thanks for the review!

What we want to express here is that the table only will show only 1000 flaky tests sorted by the number of occurrences (as we detected that a certain flaky test was detected in more commits than another flaky test)

Reply:
I think it works. Thanks!